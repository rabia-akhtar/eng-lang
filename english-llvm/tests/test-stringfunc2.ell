int main(){
	string s1 = "hello world";
	string s2 = calloc(1, 20);
	strcpy(s2, s1);
	print_string(s1);
	print_string(s2);
	free(s2);
	return(0);
}