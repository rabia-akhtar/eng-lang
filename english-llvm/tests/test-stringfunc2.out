hello world
hello world
hello worldhello world
l